src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1041,6 +1041,11 @@
 helps['monitor log-analytics workspace update'] = """
 type: command
 short-summary: Update a workspace instance
+examples:
+  - name: Update a workspace instance. (autogenerated)
+    text: |
+        az monitor log-analytics workspace update --resource-group MyResourceGroup --set retentionPolicy.days=365 --workspace-name MyWorkspace
```

Comment:
I don't think workspace has `retentionPolicy` field. It should be wrong example.

Reply:
removed